### ISSUE TITLE
Cannot run python script from within custom notebooks directory

### ISSUE THREAD

**Issue by fhg-isi** [gavincyi/jupyterlab-executor#6]
If I try to execute demo.py in my custom notebooks directory C:\python_env\workspace, 
I get following output and the file is not found:

```
Windows PowerShell
Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.

Lernen Sie das neue plattformübergreifende PowerShell kennen – https://aka.ms/pscore6

PS C:\python_env\App\WinPython\notebooks> python demo.py 
C:\python_env\App\WinPython\python-3.11.1.amd64\python.exe: can't open file 'C:\\python_env\\App\\WinPython\\notebooks\\demo.py': [Errno 2] No such file or directory
PS C:\python_env\App\WinPython\notebooks> 
```

The right path would be

C:\\python_env\\workspace\\demo.py

instead of 

C:\\python_env\\App\\WinPython\\notebooks\\demo.py

=> Where can I tell jupyter-executor the root path / path to notebooks directory?

**Comment by gavincyi** [comment]
Is `C:\python_env\App\WinPython\notebooks` your jupyter lab running directory?

**Comment by fhg-isi** [comment]
I start Jupyterlab with --notebook-dir=C:\python_env\workspace. However, the directory of the power shell is still C:\python_env\App\WinPython\notebooks.

**Comment by gavincyi** [comment]
Thanks for reporting the issue. The issue can be replicated on my end. I will release a hotfix on it in the coming days.

**Comment by gavincyi** [comment]
Fixed and released in v2023.1.1
